taxi/views.py

```diff
@@ -6,7 +6,13 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 
 from .models import Driver, Car, Manufacturer
-from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
+from .forms import (
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    CarForm,
+    CarModelSearchForm,
+    DriverUsernameSearchForm,
+    ManufacturerNameSearchForm)
```

Comment:
In case there are no imports on the same line with an opening parenthesis, your closing parenthesis should not have any imports on the same line as well. Please, resolve this issue